### ISSUE TITLE
Onboarding repository `maplibre-tile-spec`

### ISSUE THREAD

**Issue by nyurik** [maplibre/maplibre#365]
https://github.com/maplibre/maplibre-tile-spec

## Motivation
The new tile format `mlt` has been long in the making, and is now at the point when it needs an established collaborative space to move forward. `mlt` is a critical underlying specification for our progress forward.  The `mlt` repo will include the specification, tests, as well as multiple encoder and decoder implementations.

As this repo stands now, it has proven the initial goals of a POC:
* new format is significantly smaller when tested on OpenMapTiles and Microsoft MVTs
* new format does not gain significantly from generic compression (similar to jpg/png)
* new format can encode everything MVT has
* new format can reconstruct the original MVT with reasonable or even identical precision

## Acceptance
- [x] Any two board members must agree to accept a new repository.
  **Approved by:** @nyurik @HarelM 

## Licensing
- [ ] The repo license is BSD-3 or MIT.
  *Repos may allow dual-licensing under other open source licenses, e.g. MIT OR Apache.*
- [ ] The repo contains `Copyright (c) <year> MapLibre contributors` in license file(s) and in the readme.

## Special files
- [ ] `/README.md`
  *Description, link to the main maplibre.org page, name of the OSM-US Slack channel for discussions and an [invite link](https://slack.openstreetmap.us), etc*
- [ ] `/LICENSE`
   *Dual-licensed repos may have additional files like `LICENSE-MIT` and `LICENSE-APACHE`*
- [ ] `/SECURITY_POLICY.txt`
    Add the text:
    ```
    For an up-to-date policy refer to
    https://github.com/maplibre/maplibre/blob/main/SECURITY_POLICY.txt
    ```
- [ ] `/CONTRIBUTING.md`
- [ ] The repo has Pull Request and Issue Templates in `/.github` dir.
- [ ] The repo has `/.github/FUNDING.yml` file copied from [maplibre-gl-js/funding](https://github.com/maplibre/maplibre-gl-js/blob/main/.github/FUNDING.yml)
- [x] `/CODE_OF_CONDUCT.md`
  *This file should only link to our [primary code of conduct](https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md). Use this markup for consistency:*
 `# Contributor Covenant`
 `[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/maplibre/maplibre/blob/main/CODE_OF_CONDUCT.md)`

#### General page
- [x] **[Features]** Disable unused features like wiki.
- [ ] **[Features]** Enable `Sponsorships` checkbox (see also FUNDING.yaml above).
- [x] **[Features]** Enable `Preserve this repository`.
- [x] **[Pull Requests]** Community is encouraged to use `squash merge`. Disable other merge types if possible.
- [x] **[Pull Requests]** Enable `Automatically delete head branches`.

#### Access
- [x] The repo has at least one admin who is ideally not part of the Governing Board: @mactrem

#### Branches
- [x] The primary branch is named `main`.
- [ ] Set up branch rules to require CI pass and an approval before merge.
  *For smaller projects it might be OK to ignore this rule.*

## Miscelaneous
- [ ] The repo must not have any personal branches.
  *All work should be done on forks and submitted via PRs, including by the admins.*
- [ ] Repo has a proper GitHub description and an optional web site
  *Use the gear icon in the upper right corner of the repo page.*
- [ ] CI automatically runs on all pull requests before merging using GitHub actions
- [ ] Grant admin rights to the board members and automation accounts for packages <list-of-packages>
  - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder wipfli`
  - [crates.io](https://crates.io/): package settings / add owner: `nyurik klokan lseelenbinder wipfli`

## Community
- [ ] The new repo has been announced in the `#maplibre` OSMUS slack channel.
- [ ] The new repo has been announced in the next monthly meeting of the Technical Steering Committee.
- [ ] The new repo has been announced in the `@maplibre` twitter.

**Comment by HarelM** [comment]
Approved. keep up the good work!

**Comment by lseelenbinder** [comment]
Thank you, @nyurik! Looking forward to seeing the progress.

Let's proceed with onboarding but keep this issue open until we get more of the checklist done.

**Comment by nyurik** [comment]
Thanks everyone. I now have admin rights on that repo. I will wait for @mactrem to merge his working branch into `main`, and once that is done, I will transition it over to maplibre.  The next steps after that will be:

* [ ] Reorganize dir structure, e.g. top level could be `spec`, `java`, `rust`, `js`, `utils`, `fixtures` -- to allow for multi-language encoding/decoding implementations
* [ ] Introduce proper CI to build all languages, generate docs, etc
